model/channel_stats.go

```diff
@@ -4,11 +4,12 @@
 package model
 
 type ChannelStats struct {
-	ChannelId       string `json:"channel_id"`
-	MemberCount     int64  `json:"member_count"`
-	GuestCount      int64  `json:"guest_count"`
-	PinnedPostCount int64  `json:"pinnedpost_count"`
-	FilesCount      int64  `json:"files_count"`
+	ChannelId         string `json:"channel_id"`
+	MemberCount       int64  `json:"member_count"`
+	GuestCount        int64  `json:"guest_count"`
+	PinnedPostCount   int64  `json:"pinnedpost_count"`
+	HasNewPinnedPosts bool   `json:"has_new_pinned_posts"`
```

Comment:
@ashishbhate @Willyfrog it is related channel stats API, as we need to make changes in that reason being this ticket and issue due to file count

Reply:
@AshishDhama and what is the problem with making changes to the stats? Is it going to cause breaking changes? or is this affecting something else? Sorry, I'm a bit out of context here so I'm not sure what should I be looking for or finding NS.

Reply:
@Willyfrog I think we need to make sure this pr isn't adding something like File count, for which we have added param 
https://github.com/mattermost/mattermost-server/pull/22096

Reply:
Ah! Gotcha!

@sk409 once, you are done with updating the branches and fixing the conflicts, can you take a look at the above PR and not ask for the file count. That's a very expensive request in some scenarios and we should try to avoid getting it if it isn't needed.

Let us know if you need any guidance on how to avoid it.

Reply:
@Willyfrog @AshishDhama 
Can I always add exclude_files_count query parameter after SocketEvents.POST_EDITED arrives?
I‘m planning to add exclude_files_count to this request.
https://github.com/mattermost/mattermost-webapp/blob/master/actions/websocket_actions.jsx#L745

Reply:
@Willyfrog @AshishDhama 
I tried to deal with it for the time being, so please check it.